reports/final/report.tex

```diff
@@ -225,17 +225,16 @@ \subsection{Hardware} % Ethan
 time to add additional channels. The two spare channels would also have a higher
 sample rate to improve the utility of the instrument.
 
-It should be noted that all the data acquisition channels are connected to an
-FPGA rather than a microprocessor. This decision was made to ensure we had no
-real time data processing limitations and precise timing in our sampling of the
-signal waveforms. Moreover, it is common with existing commercial oscilloscopes
-to have an FPGA or ASIC fulfill these real time requirements. One compromise
-that was made in the design was the use of an FPGA module rather than installing
-the chip to the PCB directly. While such a module could not be used in TeachEE
-in a mass-production setting, it greatly shortened the PCB design time, and
-allows the FPGA to be easily replaced in the event of hardware failure.
-Moreover, the FPGA combines all the sample streams and provides a single point
-of contact to the USB interface.
+All the data acquisition channels are connected to an FPGA rather than a
+microprocessor. This was decided to ensure that there is no real-time data
```

Comment:
I think "This decision was made to" sounds better than "This was decided to"

Reply:
6351a86831140cdb8716b9f97d305a515d1f5409